iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/methods/DeviceMethodTests.java

```diff
@@ -128,74 +128,6 @@ public void serviceClientTokenRenewalWithAzureSasCredential() throws Exception
         super.invokeMethodSucceed();
     }
 
-    @Test
-    @StandardTierHubOnlyTest
-    @ContinuousIntegrationTest
-    public void invokeMethodInvokeParallelSucceed() throws Exception
```

Comment:
These doSomethingInParallel tests are not a reliable indicator of thread safety. Only unit tests can cover this scenario reliably.